readme.md

```diff
@@ -399,6 +399,23 @@ test('some test', async ({ ok, teardown }) => {
 })
 ```
 
+You can call `teardown` multiple times in a test and every function passed will be called after a test ends
```

Comment:
try to keep the same voice as the docs, the second person ("you") isn't used here